src/components/WwwFrame/SearchBar.vue

```diff
@@ -5,7 +5,7 @@
 		autocomplete="off"
 		@submit.prevent="onSubmit"
 	>
```

Comment:
Implementation example. Really the only change in most cases will be adding the fill in CSS.